src/application/entity/user.ts

```diff
@@ -0,0 +1,12 @@
+import { v4 as uuidv4 } from 'uuid';
+
+export class User {
+    public username: string;
```

Comment:
```suggestion
    public name: string;
```

userなのは自明なのでnameだけで良さそうと思ったりした！